jest.config.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const path = require('path');
+
+const esModules = ['@folio'].join('|');
+
+module.exports = {
+  collectCoverageFrom: [
+    '**/(lib|src)/**/*.{js,jsx}',
+    '!**/node_modules/**',
+    '!**/test/**',
+  ],
+  coverageDirectory: './artifacts/coverage/',
```

Comment:
```suggestion
  coverageDirectory: './artifacts/coverage-jest/',
```
https://github.com/folio-org/stripes-acq-components/blob/master/jest.config.js#L12

Reply:
cc @funkymalc is it right to use in every module? I'm not sue that sonarcloud fetches coverage from there